tf/aks/README.md

```diff
@@ -0,0 +1,85 @@
+# Azure Kubernetes Service Module
+
+This module is responsible for the deployment of Azure Kubernetes Service inside a given environment. The module is developed in its own repository [here](https://github.com/venkatsunilm/terraform-azure-ref-aks-module).
+
+The [environment-base](../environment-base/README.md) deployment has to be run before this one.
+
+# Usage
+
+Create a service principal for Azure Kubernetes Service, following [this documentation](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal).
+
+or the quick way to follow these steps to achieve the goal. 
+
+1. az account show
+
+{
+  "environmentName": "AzureCloud",
+  "homeTenantId": "3cc81ecd06f86",
+  "id": "0cea2519ddb",
+  "isDefault": true,
+  "managedByTenants": [],
+  "name": "Azure subscription venkat test",
+  "state": "Enabled",
+  "tenantId": "3cc81ecd06f86",
+  "user": {
+    "name": "ve...@hotmail.com",
+    "type": "user"
+  }
+}
```

Comment:
Please don't add your tenant details (IDs) into git files, you can change it with dummy values XXXXXXXXXX

Reply:
These are actually dummy values 😊, but yes i will replace them like you suggested.

Reply:
Replaced with dummy names.